.storybook/components/BorderRadius.js

```diff
@@ -51,14 +52,14 @@ const BorderRadiusName = styled(Text)`
   `};
 `;
 
-const BorderRadius = ({ size }) => (
+const BorderRadius = ({ size, ...props }) => (
```

Comment:
Is passing the props necessary for extracting tokens to an external package? Or are you abiding to the Boy Scout Rule?

Reply:
This is a Storybook component for internal/documentation use only. I'm spreading the props to be able to apply a custom style for the new `pill` border radius here: https://github.com/sumup-oss/circuit-ui/pull/572/files#diff-725b015fcf8c1b7bf9010aff033c7d38R180